CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### 1.0.0 Unreleased
 
+- Set inverse asoociation for cache_has_many without `:embed => true`
```

Comment:
asoociation -> association :P

Reply:
I would say it is for an "id embedded cach_has_many associations" rather than saying what it isn't for.